docs/UserGuide.md

```diff
@@ -461,18 +477,23 @@ This action is irreversible.
 
 ## FAQ
 
-**Where is my data saved?**
-NUSEatWhere's `Food Guide` automatically saves your data (including your tags and any new eateries) into a file
+**Where is my data saved?** <br>
+NUSEatWhere's `Food Guide` data (including your tags and any new eateries) is automatically saved into the file named
 `foodguide.json` in the `data` folder.
 
-**Transferring data to friends or other devices**:
+**How do I transfer data to friends or other devices?** <br>
 Make a copy of the data file (`foodguide.json` in the `data` folder) and transfer it to your friend or the other device
 by any means of file sharing (thumbdrive, email, Google Drive, etc.). In the other installation of
 `Food Guide`, replace the default data file with your data.
 
-**Manually editing `Food Guide` data**:
+**How can I manually edit `Food Guide` data?** <br>
 If you are familiar with the `json` file format, you may edit the data file `foodguide.json` directly.
```

Comment:
suggestion: "you may edit the data file `foodguide.json` directly using a text editor."